src/composables/table.ts

```diff
@@ -52,6 +53,27 @@ export interface TableActionItem<T = TableRowData> extends ActionItem {
   show?: boolean | ((rowData: T, rowIndex: number) => boolean)
 }
 
+export interface TableBulkActionItem extends ActionItem {
+  /**
+   * Whether or not the bulk action item is enabled.  Disabled actions are
+   * visible in the UI, but do not trigger click events.
+   */
+  disabled?: boolean
+  /**
+   * The callback method triggered by the action item buttons click event.
+   * @param selected the array of selected rows by the primary key `id`
+   * @param _ NOTE(spk): Holding for T[]
```

Comment:
Dumb question: leaving this spot in case we sent back an array of objects?

Reply:
Yeah, my hunch is that we will have quite a few instances where we'll want the full record at hand in the action, and it made sense to me for it to be the second argument.  Even just being able to trigger a flash that says "Successfully updated ${resouce.name}" vs "Successfully updated ${id}" will be handy.  I may try to squeeze this in yet.